redaxo/src/core/backend.php

```diff
@@ -6,6 +6,7 @@
 
 header('X-Robots-Tag: noindex, nofollow, noarchive');
 header('X-Frame-Options: SAMEORIGIN');
+header("Content-Security-Policy: frame-ancestors 'self'");
```

Comment:
Macht der Header noch was anderes, als der in der Zeile drüber?
Oder geht es darum sicher zu gehen, falls nur dieser Header unterstützt wird, oder so?

Reply:
Die header sollten feature-identisch sein. Der x-frame-options ist deprecated

Reply:
Ok, dann sollten wir das rein nehmen!

Reply:
Ich frag' mal mit meinem Halbwissen:

Müssen die Header nicht auch im Medien Manager so übernommen werden, um auch eine missbräuchliche Auslieferung von Files, die einen iframe einbetten, darüber zu verhindern? (Für den Fall der Fälle)

Reply:
media_manager kümmert sich "eigentlich" nur um die Frontenddinge, und da muss man das selbst bestimmen können. Die vereinzelten Medienaufrufe aus dem Backend sollten theroretisch auch wie alle Backendseiten reagieren, aber ich sehe hier keine Sicherheitslücke oder Problematik, vor allem, weil man sie ja auch im Frontend mit den Backendmediatypen aufrufen könnte. Hier gehte ausschliesslich um Backendseiten.